### PR TITLE
Logobar theme

### PR DESCRIPTION
**Pull request by malramsay64**
This implements the logobar theme contributed by jnothman in #5 in a more robust way.

### CODE REVIEW

**Review by jnothman — COMMENTED** [review]
I think you need some emphasis in the docs that the dots are produced by use of subsection.

I'll try this out.

**Review by jnothman — COMMENTED** [review]
This is nice. :)

I feel like the top margin is slightly excessive, particularly with aspectratio=169, but it's okay

File: beamerouterthemeusyd-logobar.sty [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+\newlength{\theme@hpadding}
+\setlength{\theme@hpadding}{1.5em}
+
+% This is the whitespace on the top and bottom of the slides.
+\newlength{\theme@vpadding}
+\setlength{\theme@vpadding}{0.5em}
+
+\defbeamertemplate*{frametitle}{usyd logo}[1][]
+{%
+  \vskip3em
```
Comment (malramsay64) [inline beamerouterthemeusyd-logobar.sty]: I think this value is the top margin you are referring to being too large. Changing this line to `\vskip2em` brings the slide title closer to the logobar.

Comment (jnothman) [reply beamerouterthemeusyd-logobar.sty]: I think this might be too large also, but I was actually referring to the margin north of the logobar

Comment (malramsay64) [reply beamerouterthemeusyd-logobar.sty]: I have made modifications compressing all the top and bottom components. Is that more appropriate?
screen shot 2018-06-18 at 18 59 37

**Comment by jnothman** [comment]
I'm not sure about this... It's now a bit too dense at the top for visual
aesthetics...​ Tricky.

**Comment by malramsay64** [comment]
I'm going to go with splitting the difference between the two. If in using the template you feel quantities need tweaking I am open to minor changes.

**Comment by jnothman** [comment]
LGTM!

**Comment by jnothman** [comment]
Great, thanks!​

**Pull request merged**